src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java

```diff
@@ -195,7 +197,6 @@ public void testMiltipleConditions() throws Exception {
         archiver = (ArtifactArchiver)conditionalPublisher2.getPublisher();
         assertEquals("**/*.jar", archiver.getArtifacts());
         assertEquals("some/bad.jar", archiver.getExcludes());
-        assertTrue(archiver.isLatestOnly());
```

Comment:
(`isLatestOnly` is deprecated and not preserved for `configRoundTrip`)
It makes sense to replace with `assertBataBoundBeansEqual`.